dbus_service.py

```diff
@@ -601,20 +599,40 @@ def get_values_for_inverter(self):
             current = get_ahoy_field_by_name(meter_data, self.pvinverternumber, "I_AC")
 
         elif self.dtuvariant == constants.DTUVARIANT_OPENDTU:
-            root_meter_data = meter_data["inverters"][self.pvinverternumber]
+            if "AC" in meter_data["inverters"][self.pvinverternumber]:
+                root_meter_data = meter_data["inverters"][self.pvinverternumber]
+                old_firmware=True
```

Comment:
Oder besser: Die Var. nicht "old" nennen. Eher "firmware_v123_or_newer". Das ist auch in 2 Jahren noch lesbar und klappt auch, wenn es nochmal einen breaking change gibt.

Reply:
firmware_v24_2_12_or_newer ist jetzt in Verwendung